main.rb

```diff
@@ -59,31 +56,29 @@ def welcome_screen
   welcome = "*"*welcome.length + "\n" + welcome + "\n" + "*"*welcome.length
 end
```

Comment:
This is very tricky. Why not just one line with string interpolation?

Reply:
Resolved